draft-ietf-shmoo-remote-fee.md

```diff
@@ -103,6 +103,9 @@ This document rather states that if remote participation is provided,
 there should always be a free option to make the process as open as possible. Having said that,
 it is of course strongly anticipated that at least all working group sessions as well as BoFs
 and the administrative plenary of an IETF meeting provide an option for remote participation.
+Stated another way, this document defines a *principle* and leaves the
+*mechanism* of it to those with fiscal, and related, resopnsibilities
+such as the IETF LLC.
```

Comment:
```suggestion
```
Removing this as I find the "such as" part actually a bit vague... Also I'm generally not a bit far of "stated another way" statement; It's better to get it right the first time ;-)

But keeping the editorial change of the first sentence in the PR and will merge!